private-templates-service/client/src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { RootState } from '../../store/rootReducer';
+import { UserType } from '../../store/auth/types';
+import { UserAvatar } from './UserAvatar';
+import mainLogo from '../../assets/adaptive-cards-100-logo.png';
+
+// CSS
+import { OuterSideBarWrapper, LogoWrapper, UserWrapper, Name, Title, NavMenu, MainItems, SignOut, Logo, LogoTextWrapper, LogoTextHeader, LogoTextSubHeader } from './styled';
+
+
+
+interface Props {
+  authButtonMethod: () => void;
+  isAuthenticated: boolean;
+  user?: UserType;
+}
+
+const mapStateToProps = (state: RootState) => {
+  return {
+    isAuthenticated: state.auth.isAuthenticated,
+    user: state.auth.user,
+  }
+}
+
+const navMenuLinks = [{
```

Comment:
Could this be simplified to just an array of objects, rather than an array of objects, of array of objects?

Reply:
Unfortunately this is the expected input for the component that we are using from fabric-ui.